doc/user/RouteRequest.md

```diff
@@ -29,7 +29,7 @@ and in the [transferRequests in build-config.json](BuildConfiguration.md#transfe
 | locale                                                                                                       |        `locale`        | TODO                                                                                                                                                     | *Optional* | `"en_US"`        |  2.0  |
 | [maxDirectStreetDuration](#rd_maxDirectStreetDuration)                                                       |       `duration`       | This is the maximum duration for a direct street search for each mode.                                                                                   | *Optional* | `"PT4H"`         |  2.1  |
 | [maxJourneyDuration](#rd_maxJourneyDuration)                                                                 |       `duration`       | The expected maximum time a journey can last across all possible journeys for the current deployment.                                                    | *Optional* | `"PT24H"`        |  2.1  |
-| modes                                                                                                        |        `string`        | The set of access/egress/direct/transit modes to be used for the route search.                                                                           | *Optional* | `"TRANSIT,WALK"` |  2.0  |
+| modes                                                                                                        |        `string`        | The set of access/egress/direct/transfer modes (separated by a comma) to be used for the route search.                                                   | *Optional* | `"WALK"`         |  2.0  |
```

Comment:
While testing some changes I noticed we don't actually support providing default transit modes in config.